query-service-impl/src/main/java/org/hypertrace/core/query/service/ExecutionContext.java

```diff
@@ -137,12 +139,12 @@ private ColumnMetadata toColumnMetadata(Expression expression) {
     ValueCase valueCase = expression.getValueCase();
     switch (valueCase) {
       case COLUMNIDENTIFIER:
-        ColumnIdentifier columnIdentifier = expression.getColumnIdentifier();
-        String alias = columnIdentifier.getAlias();
+      case ATTRIBUTE_EXPRESSION:
+        String alias = getAlias(expression);
         if (alias != null && alias.trim().length() > 0) {
```

Comment:
nit: not possible for this code to return null. Below would be semantically equivalent

```suggestion
        if (!alias.isBlank()) {
```